app/screens/reschedule_draft/reschedule_draft.tsx

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback, useRef, useState} from 'react';
+import {useIntl} from 'react-intl';
+import {Keyboard, SafeAreaView, StyleSheet, View} from 'react-native';
+
+import {updateScheduledPost} from '@actions/remote/scheduled_post';
+import Loading from '@components/loading';
+import {useServerUrl} from '@context/server';
+import {useTheme} from '@context/theme';
+import DatabaseManager from '@database/manager';
+import useAndroidHardwareBackHandler from '@hooks/android_back_handler';
+import useNavButtonPressed from '@hooks/navigation_button_pressed';
+import {usePreventDoubleTap} from '@hooks/utils';
+import DateTimeSelector from '@screens/custom_status_clear_after/components/date_time_selector';
+import PostError from '@screens/edit_post/post_error';
+import {buildNavigationButton, dismissModal, setButtons} from '@screens/navigation';
+import {logDebug} from '@utils/log';
+import {changeOpacity} from '@utils/theme';
+import {getTimezone} from '@utils/user';
+
+import type ScheduledPostModel from '@typings/database/models/servers/scheduled_post';
+import type {AvailableScreens} from '@typings/screens/navigation';
+import type {Moment} from 'moment-timezone';
+
+type Props = {
+    currentUserTimezone?: UserTimezone | null;
+    componentId: AvailableScreens;
+    closeButtonId: string;
+    draft: ScheduledPostModel;
+}
+
+const OPTIONS_PADDING = 12;
+
+const styles = StyleSheet.create({
+    body: {
+        flex: 1,
+    },
+    container: {
+        flex: 1,
+    },
```

Comment:
there is no point of having two styles that do the same.. keep one, remove the other one